build-showmewebcam.sh

```diff
@@ -9,35 +11,36 @@ if [ ! -d "$BUILDROOT_DIR" ]; then
 fi
 
 export BOARDNAME=$1
+shift
 
 case "$BOARDNAME" in
   raspberrypi0)
-  ;;
-  raspberrypi0w)
-  ;;
+    ;;
+  raspberrypi0_2)
+    ;;
   raspberrypi4)
-  ;;
+    ;;
   *)
-    echo "usage: BUILDROOT_DIR=buildroot $0 (boardname)"
-    echo "boardname: raspberrypi0, raspberrypi0w, raspberrypi4"
+    echo "usage: BUILDROOT_DIR=buildroot $0 (boardname) [component]"
+    echo "boardname: raspberrypi0, raspberry0_2, raspberrypi4"
     exit 1
   ;;
 esac
 
+# Add overridden buildroot packages
+cp -R package-override/* "$BUILDROOT_DIR/package"
+
 # Merge custom buildroot configurations
 CONFIG_="BR2" KCONFIG_CONFIG="configs/${BOARDNAME}_defconfig" "$BUILDROOT_DIR/support/kconfig/merge_config.sh" -m -r configs/config "configs/$BOARDNAME"
 sed "1i ### DO NOT EDIT, this file was automatically generated\n" -i "configs/${BOARDNAME}_defconfig"
 
-# Merge kernel configurations
-if [ -f "board/linux-${BOARDNAME}.config" ]; then
-  KCONFIG_CONFIG="board/linux.config" "$BUILDROOT_DIR/support/kconfig/merge_config.sh" -m -r board/linux-base.config "board/linux-${BOARDNAME}.config"
-else
-  cp board/linux-base.config board/linux.config
```

Comment:
Right here we could just use the buildroots kernel config for the board as a base.